CONTRIBUTING.md

```diff
@@ -138,13 +138,29 @@ npm version <major|minor|patch> -m "## 1.2.3 - 2017-01-13
 
 - Change included in this release
 - Another change included in this release"
+```
+
+Then run:
+```bash
 git push --tags --set-upstream origin refs/heads/v1.2.3:refs/heads/v1.2.3
 ```
 
+Open a PR for the release version.
+
 To actually publish, you will need access to an `npm` account that owns `accessible-autocomplete`. Merge the version PR and then run:
 
 ```bash
 git checkout master
 git pull --rebase
 npm publish
-```
\ No newline at end of file
+```
+
+Create a release in the [Github interface](https://github.com/alphagov/accessible-autocomplete/releases/new)
+  - select the latest tag version
+  - set "v[version-number]" as the title
```

Comment:
Not a deal-breaker, but in Frontend docs we've used this style for placeholders:

v\<VERSION-NUMBER\>